pkg/exporter/probe/legacy.go

```diff
@@ -12,9 +14,64 @@ import (
 var legacyMetricsLabels = []string{"target_node", "target_namespace", "target_pod", "node", "namespace", "pod"}
 var StandardMetricsLabels = []string{"k8s_node", "k8s_namespace", "k8s_pod"}
 var TupleMetricsLabels = []string{"protocol", "src", "src_type", "src_node", "src_namespace", "src_pod", "dst", "dst_type", "dst_node", "dst_namespace", "dst_pod", "sport", "dport"}
+var AdditionalLabelValueExpr []string
 
 func BuildStandardMetricsLabelValues(entity *nettop.Entity) []string {
-	return []string{nettop.GetNodeName(), entity.GetPodNamespace(), entity.GetPodName()}
+	metaPodLabels := []string{nettop.GetNodeName(), entity.GetPodNamespace(), entity.GetPodName()}
+	return append(metaPodLabels, BuildAdditionalLabelsValues(entity)...)
+}
+
+func InitAdditionalLabels(additionalLabels []string) error {
+	if len(additionalLabels) == 0 {
+		return nil
+	}
+
+	//append to StandardMetricsLabels and AdditionalLabelValueExpr
+	for additionalKV := range additionalLabels {
+		labelKVPair := strings.Split(additionalLabels[additionalKV], "=")
+		StandardMetricsLabels = append(StandardMetricsLabels, strings.TrimSpace(labelKVPair[0]))
+		AdditionalLabelValueExpr = append(AdditionalLabelValueExpr, strings.TrimSpace(labelKVPair[1]))
+	}
+
+	return nil
+}
+
+func BuildAdditionalLabelsValues(entity *nettop.Entity) []string {
```

Comment:
Please add an unittest for this function

Reply:
ok, I completed a unit test